### PR TITLE
Override fishing alert

### PR DESCRIPTION
**Pull request by anxolin**
Fixed the warning on top

Before:
image

After:
IMG_8731

### CODE REVIEW

File: src/custom/components/Header/URLWarning/index.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { PRODUCTION_URL } from '@src/custom/constants'
```
Comment (W3stside) [inline src/custom/components/Header/URLWarning/index.tsx:1]: `@src`? why?

File: src/custom/components/Header/URLWarning/index.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+import React from 'react'
+import { default as URLWarningUni } from './URLWarningMod'
```
Comment (W3stside) [inline src/custom/components/Header/URLWarning/index.tsx:3]:
```suggestion
import URLWarningUni from './URLWarningMod'
```

File: src/pages/App.tsx [hunk 3]
```diff
@@ -5,7 +5,7 @@ import GoogleAnalyticsReporter from 'components/analytics/GoogleAnalyticsReporte
 import AddressClaimModal from '../components/claim/AddressClaimModal'
 import Header from 'components/Header'
 import Polling from '../components/Header/Polling'
-import URLWarning from '../components/Header/URLWarning'
+import URLWarning from '@src/custom/components/Header/URLWarning/index'
```
Comment (W3stside) [inline src/pages/App.tsx:8]: `@src` why?

File: src/custom/components/Header/URLWarning/index.tsx (hunk 1 above)
Comment (anxolin) [reply src/custom/components/Header/URLWarning/index.tsx:1]: good catch, it was vscode auto-importing

**Review by W3stside — APPROVED** [review]

**Comment by dustpan28645** [comment]
Be honest was right is right what's wrong is wrong !!!!💯
tipsAreWelcome
***in**filecoin f1j4oqo6bf5ru6fnpf2qqmyj3ecejtz7jjelgntva

**Pull request merged**